.github/workflows/live_tests.yml

```diff
@@ -86,6 +86,17 @@ jobs:
             echo "USE_LOCAL_CDK_FLAG=" >> $GITHUB_ENV
           fi
 
+      - name: Setup Test Evaluation Mode using Support Level
```

Comment:
Changing the CI flow according to metadata content have always been handled from inside airbyte-ci so far. Is there a strong reason for not doing it there?

Reply:
I did it here for visibility - since the command line options given to the `subcommand` in the workflow step are so similar to the command we give to pytest, I thought it would be nice to have it here instead of in airbyte-ci where it's somewhat hidden. But since putting it in airbyte-ci is the convention I can do that.